ntp.go

```diff
@@ -333,8 +328,8 @@ func getTime(host string, opt QueryOptions) (*msg, ntpTime, error) {
 	m.setVersion(opt.Version)
 	m.setLeapIndicator(LeapNotInSync)
 
-	xmt_time := time.Now()
-	xmt := toNtpTime(xmt_time)
+	xmtTime := time.Now()
+	xmt := toNtpTime(xmtTime)
 	m.TransmitTime = xmt
```

Comment:
I suggest replacing

```go
xmt := toNtpTime(xmtTime)
m.TransmitTime = xmt
```

with 

```go
m.TransmitTime = toNtpTime(xmtTime)
```

Since `xmt` isn't used anywhere else.

Reply:
`xmt` is used later in `m.OriginTime != xmt`